apps/next-evans/next.config.js

```diff
@@ -27,6 +27,9 @@ const nextConfig = {
       },
     ];
   },
+  eslint: {
+    ignoreDuringBuilds: true,
+  },
```

Comment:
i think both of these edits were stuff we added while troubleshooting and experimenting, not stuff we wanted long term.

that easily may not have been clear to you... but if it was, these are the sorts of things you should catch (and maybe some of the process.stdout logging above) if when you do your PR self review 👍

Reply:
Gotcha, I missed that